app-editors/visual-studio-code-bin/visual-studio-code-bin-1.47.3.ebuild

```
@@ -0,0 +1,45 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop eutils pax-utils
+
+DESCRIPTION="Multiplatform Visual Studio Code from Microsoft"
+HOMEPAGE="https://code.visualstudio.com"
+SRC_URI="https://update.code.visualstudio.com/${PV}/linux-x64/stable -> ${P}.tar.gz"
+
+RESTRICT="mirror strip"
+
+LICENSE="MIT Microsoft-VSCode"
+SLOT="0"
+KEYWORDS="~amd64"
```

```suggestion
KEYWORDS="-* ~amd64"
```
because it's a binary and can't work [elsewhere](https://devmanual.gentoo.org/keywording/index.html)